tests/Functional/SiteBrandingBlockTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
```

Comment:
Don't use a functional test just to check some classes and/or markup.
Please use something similar to https://github.com/openeuropa/oe_theme/blob/2.x/tests/Kernel/RenderingTest.php#L103

Reply:
in the end that one was incompatible with blocks